content/3_Silver/Binary_Search.problems.json

```diff
@@ -77,6 +77,18 @@
         "usacoId": "419"
       }
     },
+    {
+      "uniqueId": "usaco-1159",
+      "name": "Connecting Two Barns",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1159",
+      "source": "Gold",
```

Comment:
is this silver or gold? also not sure about adding this problem to all these modules (haven't looked at the problem) because problems should only be added if they add something that the other problems don't cover

Reply:
oh oops, should be silver

Reply:
> problems should only be added if they add something that the other problems don't cover

ok. ill remove it from dsu, but i think it should stay in binary search because there's not a lot of graph problems? (lmk if you think it should be removed)

Reply:
should i remove it from dfs as well?

Reply:
not too sure abt dfs?

Reply:
i think it's fine rn for dfs (not a lot of minimizing cost/tp problems), so il resolve this thread, but feel free to reopen it if you have a diff opinion :)